src/component/SettingsForm/index.jsx

```diff
@@ -1,15 +1,19 @@
+/* eslint-disable react-hooks/exhaustive-deps */
 import React, { useState } from 'react';
 import Form from 'react-bootstrap/Form';
 import Button from 'react-bootstrap/Button';
 import PropTypes from 'prop-types';
 import BootstrapSwitchButton from 'bootstrap-switch-button-react';
-import { PERIODS } from '../../services/timer';
+import firebase from '../../firebase';
 
-const SettingsForm = ({ onShouldAutoStartChange, shouldAutoStart }) => {
-  const [periods, setPeriods] = useState(PERIODS);
+const SettingsForm = ({ onShouldAutoStartChange, shouldAutoStart, dataOfPeriods }) => {
+  const [periods, setPeriods] = useState(dataOfPeriods);
   const handleSubmit = (e) => {
     e.preventDefault();
-    Object.assign(PERIODS, periods);
+    Object.assign(dataOfPeriods, periods);
```

Comment:
It's necessary because it keeps values appear in settings form when I close it and return back to it, the values I changed still there. this line is necessary ``Object.assign(periodsDB, periods)`` 
`` Object.assign(PERIODS, periods);`` this line I deleted @mujz